source/manual/test-and-build-a-project-with-github-actions.html.md

```diff
@@ -145,20 +138,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@v1
         with:
+          bundler-cache: true
           ruby-version: ${{ matrix.ruby }}
-      - uses: actions/cache@v1
-        with:
-          path: vendor/bundle
-          key: bundle-${{ matrix.ruby }}-${{ hashFiles('**/*.gemspec') }}
-          restore-keys: bundle-${{ matrix.ruby }}
-      - run: bundle install --jobs 4 --retry 3 --path vendor/bundle
       - run: bundle exec rake
 ```
 
 Notes:
 
-- The [deployment][bundler-deployment] flag for bundler cannot be used for gems
-  as they should not have a Gemfile.lock committed.
```

Comment:
Ah it's great that the action seems to take care of this automatically now: https://github.com/ruby/setup-ruby/blob/e5b6990d5186f1ac1d8074e46b72f9efb63aa305/index.js#L256

Reply:
Thanks, I've fixed a trailing space issue & added this link in the commit :). Will also check out the Yarn discussion!